pywinauto/base_wrapper.py

```diff
@@ -48,6 +50,8 @@
 from time import sleep
 from .actionlogger import ActionLogger
 from .mouse import _get_cursor_pos
+from .timings import TimeoutError
```

Comment:
It's good to group all `.timings` imports close to each other. Also please do separate import statement for each name like so:
```python
from .timings import wait_until
from .timings import Timings
...
```